### DISCUSSION TITLE
Welcome to PSO-for-VRPTW Discussions!

### DISCUSSION THREAD

**Discussion by ByskyXie** [ByskyXie/PSO-for-VRPTW#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zszhra** [comment]
hi sir, may i ask you about this code? my thesis using PSO to solve CVRP. and I intend to ignore the time constraint here by turn it into 24 hours. but i am quiet confused with the procedure that you used while updating velocity and dot position, there are no update for those? but i found a similar equation as updating velocity, sum_factor=self.omega+c1+c2. but, if im not mistaken, the equation that i know for update the velocity in PSO is v=(omega*v0)+c1r1(Pbest-x0)+c2r2(Gbest-x0). with x0 means the first position for each particle, c1 is the value of constanta acceleration, and r1 is a random number. can you help me with this matter, please sir?
i highly appreciate it, if you can reply this month. thank you so much.
